app/markets/Bet.tsx

```diff
@@ -107,3 +134,12 @@ export const Bet = ({ fixedProductMarketMaker }: BetProps) => {
 
   return <div className="p-8 text-center text-md">Market is currently closed.</div>;
 };
+
+const Info = ({ children }: PropsWithChildren) => (
+  <Tooltip>
+    <TooltipTrigger>
+      <Icon name="info" size={16} />
+    </TooltipTrigger>
+    <TooltipContent>{children}</TooltipContent>
+  </Tooltip>
+);
```

Comment:
This Tootlip has a transparency, I think we should make it full opaque so the user can't see through it

![Screenshot 2024-09-17 at 10 08 24](https://github.com/user-attachments/assets/5583b7d4-d8ee-433d-9aea-65fc394aa8fb)

Reply:
Fixed